dom0/runner.py

```diff
@@ -89,6 +90,19 @@ def shutdown_sd_vms(self):
             if vm.is_running():
                 self.logging.info(f"Shutting down {vm.klass}: {vm.name}")
                 vm.shutdown(force=True)
+        # Wait for all VMs to shut down
+        waited = 0
+        while any(vm.is_running() for vm in sdw_vms):
+            time.sleep(1)
+            waited += 1
+            if waited >= 60:
+                msg = "Timed out waiting for SecureDrop Workstation VMs to shut down"
```

Comment:
I copied this part from the `run_cmd()` function below, can refactor into common code instead if that's preferred.